pkg/subscribe/debouncer_test.go

```diff
@@ -18,7 +18,9 @@ func TestDebouncer(t *testing.T) {
 	count := 5
 	eventsCh := make(chan types.APIEvent, count)
 	for range count {
-		eventsCh <- types.APIEvent{}
+		eventsCh <- types.APIEvent{
+			Revision: "111",
```

Comment:
If we want to simulate this properly, each event would end up having a different resource version. Because Kubernetes basically increases a number whenever the state of etcd changes.

I think the tests should reflect that if possible.